src/cli/cli.rs

```diff
@@ -495,4 +495,22 @@ pub fn build_cli() -> App<'static, 'static> {
                 .long("force-color")
                 .help("Alias for --mode=24bit")
         )
+        .arg(
+            Arg::with_name("color-picker")
+                .long("color-picker")
+                .takes_value(true)
+                .possible_values(&[
```

Comment:
It would be great if we could compute this list of possible values from the list of colorpickers in `src/cli/colorpicker.rs`.

I guess we would have to move the `tools` list out into a global `static` variable (which we would probably initialize with `lazy_static`).

Reply:
you're absolutely right, I've tried that approach before submitting the PR, but the problem is that the `tools` list would have to be available also at build time in order to generate shell completions. This would mean that `ColorPickerTool` would have to be exposed too. I figured it was not worth the extra complexity of having another file to be included in build.rs just for this. In particular because I noticed that in the help message of `pick` there's already the list of known color pickers which has to be updated every time a new one is added.

Anyway, I'll try to come up with a simple solution.

Reply:
Ok, I came up with a possible solution which is not pretty, but not too bad I think. Let me know what you think.